integration-tests/resources/jobs/tests/csv_to_pg_batch_with_invalid.yaml

```diff
@@ -0,0 +1,26 @@
+input:
+  uses: files.read_csv
+  with:
+    file: batch_with_invalid.csv
+    batch_size: 2500
+steps:
+  - uses: add_field
+    with:
+      field: full_name
+      language: jmespath
+      expression: concat([capitalize(fname), ' ' , capitalize(lname)])
```

Comment:
`expression: concat([capitalize(fname), ' ', capitalize(lname)])`

Reply:
✅ It is not necessary. Removed the file.